src/website/src/app/documentation/demos/datepicker/demos/datepicker-api.demo.html

```diff
@@ -37,11 +38,24 @@ <h4>Min/Max attributes</h4>
 <h5>Only Min</h5>
 <form clrForm clrLayout="vertical">
     <clr-date-container>
-        <input type="date" clrDate name="minDemo" [(ngModel)]="demo" min="2019-11-11">
+        <label for="minDemo">Min Demo</label>
+        <input type="date" clrDate name="minDemo" [(ngModel)]="demo" min="2019-11-17">
     </clr-date-container>
 </form>
 <clr-code-snippet [clrCode]="minExample"></clr-code-snippet>
 <h5>Only Max</h5>
+<form clrForm clrLayout="vertical">
+    <clr-date-container>
+        <label for="maxDemo">Max Demo</label>
```

Comment:
Let's also put the min/max dates in the label so people know what it is more easily.

Reply:
  ok